src/Plugin/DoctrineORMPlugin/ResetClosedEntityManagers.php

```diff
@@ -2,7 +2,7 @@
 
 namespace LongRunning\Plugin\DoctrineORMPlugin;
 
-use Doctrine\Common\Persistence\ManagerRegistry;
+use Symfony\Bridge\Doctrine\ManagerRegistry;
```

Comment:
I think this should be `\Doctrine\Persistence\ManagerRegistry` instead.

Reply:
That's my fault. You're right.